frontend/src/pages/WorkspaceTabs/WorkspaceTabs.tsx

```diff
@@ -36,8 +36,8 @@ export const WorkspaceTabs = () => {
 	const location = useLocation()
 	const navigate = useNavigate()
 
+	const { workspace_id: workspaceId } = useParams<{ workspace_id: string }>()
 	const workspaceMatch = useMatch('/w/:workspace_id/:page_id')
```

Comment:
The problem is that this match doesn't work without the `page_id` segment of the route and it returns `null`. We then get redirected to `/w/undefined/team`, but that causes the [`DefaultWorkspaceRouter` to be rendered](https://github.com/highlight/highlight/blob/dc126bcc276dc81b5f218e8ca602c6fb0df878e1/frontend/src/routers/AppRouter/AppRouter.tsx#L228) and [redirect them back to `/w/:workspace_id`](https://github.com/highlight/highlight/blob/dc126bcc276dc81b5f218e8ca602c6fb0df878e1/frontend/src/routers/OrgRouter/DefaultWorkspaceRouter.tsx#L37).

Reply:
aha! awesome find chris, thank you 🎉 would another option be to use optional params eg. `"errors/:error_secure_id?/:error_tab_key?/:error_object_id?"`? trying that locally doesn't work for me so the `useParams` is changing the logic in a way that fixes this.

Reply:
Yeah, I tried that locally as well, but I couldn't get it to work with `useMatch`. Looking at it again, I'm guessing we could get it working if we changed the route definition [here](https://github.com/highlight/highlight/blob/e2f78ab275dfd2d7039dade1dcf6d234ef52e109/frontend/src/routers/OrgRouter/WorkspaceRouter.tsx#L109), but I was kind of rushing to get the fix out.